.codeclimate.yml

```diff
@@ -37,6 +37,7 @@ engines:
     exclude_paths:
     - 'spec/**/*'
     - 'db/migrate/*'
+    - 'lib/i18n_override.rb'
```

Comment:
It's preferable to disable specific Reek offenses rather than an entire file. Could we remove this exclusion so we can see what the offenses are, and work together to see if we can resolve them?